src/implementations/config/helpers.ts

```diff
@@ -142,3 +180,14 @@ export const validate = (forms: Form[]): boolean => {
   const anyInvalidForm = !isValidForm.some((validForm: boolean) => validForm === false);
   return anyInvalidForm;
 };
+
+export const getNetworkId: {
```

Comment:
networkCodes might be a better name, get implies that u r invoking some function
also u think this will be used and can be placed in networks.ts file?

Reply:
ok updated